server/datastore/mysql/operating_system_vulnerabilities.go

```diff
@@ -50,13 +50,10 @@ func (ds *Datastore) ListVulnsByOsNameAndVersion(ctx context.Context, name, vers
 				MIN(software_cve.created_at) created_at
 			FROM
 				software_cve
-				JOIN software ON software.id = software_cve.software_id
-				JOIN software_titles ON software_titles.id = software.title_id
-				JOIN host_software ON host_software.software_id = software.id
-				JOIN host_operating_system ON host_operating_system.host_id = host_software.host_id
-				JOIN operating_systems ON operating_systems.id = host_operating_system.os_id
+				JOIN kernel_host_counts ON kernel_host_counts.software_id = software_cve.software_id
+				JOIN operating_systems ON operating_systems.os_version_id = kernel_host_counts.os_version_id
```

Comment:
We [may need to filter by team here](https://fleetdm.slack.com/archives/C086V2QK76X/p1755104568370619?thread_ts=1755020634.634219&cid=C086V2QK76X) (which will involve changing the signature of this fn to include a `*uint` for teamID).